File: rasterizer-dotnet-examples-csharp/PngWindowsPalette.cs

using ceTe.DynamicPDF.Rasterizer;

namespace rasterizer_dotnet_csharp
{
    public class PngWindowsPalette
    {
        public static void Run()
        {
            PdfRasterizer rasterizer = new PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"));
            PngIndexedColorFormat pngIndexedColorFormat = new PngIndexedColorFormat(Palette.WindowsPalette);
            PngImageFormat pngIndexed = new PngImageFormat(pngIndexedColorFormat);
            rasterizer.Draw(Program.GetOutputDocPath("png-windows-color-palette-output.png"), pngIndexed, ImageSize.Dpi72);
        }


    }
}

File: rasterizer-dotnet-vb/PngWindowsPalette.vb
Imports ceTe.DynamicPDF.Rasterizer

Public Class PngWindowsPalette
    Public Shared Sub Run()
        Dim rasterizer As New PdfRasterizer(Program.GetResourcePath("DocumentA.pdf"))
        Dim pngIndexedColorFormat As New PngIndexedColorFormat(Palette.WindowsPalette)
        Dim pngIndexed As New PngImageFormat(pngIndexedColorFormat)
        rasterizer.Draw(Program.GetOutputDocPath("png-windows-color-palette-output.png"), pngIndexed, ImageSize.Dpi72)
    End Sub
End Class

